### PR TITLE
update ode tolerance doc

### PR DESCRIPTION
**Pull request by yizhang-yiz**
#### Submission Checklist

- [x] Builds locally 
- [x] Declare copyright holder and open-source license: see below

#### Summary
Address #282. @jtimonen @caesoma @wds15 @bbbales2 

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Metrum Research Group

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

**Comment by bbbales2** [comment]
Will get it tomorrow!

**Comment by yizhang-yiz** [comment]
I try to keep it as concise as possible without distorting the implications of rtol & atol.

File: src/stan-users-guide/odes.Rmd [hunk 1]
```diff
@@ -354,25 +354,26 @@ indiscriminately stepping over an important feature of the solution.
...
-$y=(y_1,\dots,y_n)^T$, the
-solver controls estimated local error $e=(e_1,\dots,e_n)^T$ at
-certain step through its weighted root-mean-square norm $\|\cdot\|$
+$y=(y_1,\dots,y_n)^T$, at every step the
+solver controls estimated local error $e=(e_1,\dots,e_n)^T$ through its weighted root-mean-square norm
+(@SerbanHindmarsh:2005, @hairer:1993)
 
 $$
-\|e\| = \sqrt{\sum_{i=1}^n{\frac{e_i^2}{\text{relative\_tolerance}\times y_i + \text{absolute\_tolerance}}}} < 1,
+\|e\| = \sqrt{\sum_{i=1}^n{\frac{1}{n}\frac{e_i^2}{(\text{RTOL}\times y_i + \text{ATOL})^2}}} < 1
```
Comment (bbbales2) [inline src/stan-users-guide/odes.Rmd:364]: I do not like the combined definition/inequality. Separate these, or just do the inequality.

File: src/stan-users-guide/odes.Rmd [hunk 2]
```diff
@@ -354,25 +354,26 @@ indiscriminately stepping over an important feature of the solution.
...
 ### Tolerance {-}
 
-The relative and absolute tolerance control the accuracy of the
+The relative tolerance RTOL and absolute tolerance ATOL control the accuracy of the
 numerical solution. Specifically, when solving an ODE with unknowns
-$y=(y_1,\dots,y_n)^T$, the
-solver controls estimated local error $e=(e_1,\dots,e_n)^T$ at
-certain step through its weighted root-mean-square norm $\|\cdot\|$
+$y=(y_1,\dots,y_n)^T$, at every step the
+solver controls estimated local error $e=(e_1,\dots,e_n)^T$ through its weighted root-mean-square norm
```
Comment (bbbales2) [inline src/stan-users-guide/odes.Rmd:360]: The quantity below sums over states -- do we control local error as a vector or do we just control the quantity `|e|`? Clarify this (or tell me I'm misunderstanding something).

**Review by bbbales2 — CHANGES_REQUESTED** [review]
Couple of clarifications

File: src/stan-users-guide/odes.Rmd (hunk 1 above)
Comment (yizhang-yiz) [reply src/stan-users-guide/odes.Rmd:364]: Good point, on it.

File: src/stan-users-guide/odes.Rmd (hunk 2 above)
Comment (yizhang-yiz) [reply src/stan-users-guide/odes.Rmd:360]: We control `|e|`.

Comment (bbbales2) [reply src/stan-users-guide/odes.Rmd:360]: Okay cool make it like that in the text then.

Comment (yizhang-yiz) [reply src/stan-users-guide/odes.Rmd:360]: Now I see what you mean. We're at natural language limit here. We do **control** `e`, by limiting its norm. The wording here is the same as in CVODE user guide (p.24) and I'm assuming it's generally accepted in numerical analysis world.

Comment (bbbales2) [reply src/stan-users-guide/odes.Rmd:360]: Oh, I didn't read the part after the dollars. Makes sense now.

**Review by bbbales2 — APPROVED** [review]

**Pull request merged**